analyze_model_errors.py

```
@@ -0,0 +1,309 @@
+
+    # Load predictions for all splits
+    splits = []
+    all_stats = {}
+
+    for split in ['train', 'validation', 'test']:
+        try:
+            df = load_predictions(run_dir, model, split)
+            splits.append(split)
+
+            # Analyze errors
+            stats, fp, fn, tp, tn = analyze_errors(df, split)
```

Ruff (0.14.4)

Unpacked variable `tp` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)